internal/injector/config/builtin.go

```diff
@@ -38,7 +39,7 @@ var builtIn = configGo{
 				ID:             "built.WithOrchestrionVersion",
 				TracerInternal: true, // This is safe to apply in the tracer itself
 				JoinPoint: join.AllOf(
-					join.ValueDeclaration(join.MustTypeName("string")),
+					join.ValueDeclaration(typed.MustTypeName("string")),
```

Comment:
Very, very optional: This could be a global variable or functor in the `typed` package.